bin/Powheg/runcmsgrid_powheg.sh

```diff
@@ -373,8 +373,8 @@ if [ -s pwg-stat.dat ]; then
   tail -${tail} cmsgrid_final.lhe                           >  cmsgrid_final.lhe_tail
   head -${head} cmsgrid_final.lhe                           >  cmsgrid_final.lhe_F
   proclin=`expr $head + 1`
-  PROCESS=`sed -n -e ${proclin}p  cmsgrid_final.lhe |  awk '{print $4}'`
-  echo "  "$XSECTION"   "$XSECUNC"  1.00000000000E-00 "$PROCESS >>  cmsgrid_final.lhe_F
+  proc=`sed -n -e ${proclin}p  cmsgrid_final.lhe |  awk '{print $4}'`
```

Comment:
thanks @yuanchao now it makes sense, is the "-" in the variable name, which is not supported by bash.
to run the GenXsecAnalyzer you can use 
https://github.com/cms-sw/genproductions/blob/master/test/calculateXSectionAndFilterEfficiency/genXsec_cfg.py
and run it with the command
cmsRun genXsec_cfg.py inputFiles="file.root" maxEvents=1000
further instructions here
https://twiki.cern.ch/twiki/bin/viewauth/CMS/HowToGenXSecAnalyzer